manifests/bridge-marker.yml.in

```diff
@@ -16,6 +16,8 @@ spec:
         tier: node
         app: bridge-marker
         name: bridge-marker
+      annotations:
+        description: Bridge marker pod marks network bridges available on nodes as node resources
```

Comment:
Bridge marker exposes network bridges available on nodes as node resources

So we don't mark mark

Reply:
ack,
Done